src/electron/DEPS

```diff
@@ -147,6 +148,20 @@ hooks = [
       'import os, subprocess; os.chdir(os.path.join("src", "electron")); subprocess.check_call(["python3", "script/lib/npx.py", "yarn@' + (Var("yarn_version")) + '", "install", "--frozen-lockfile"]);',
     ],
   },
+  {
+    'name': 'download_libreofficekit',
+    'pattern': 'src/electron',
+    'action': [
+      'python3',
+      'src/electron/script/download-libreofficekit.py',
+      '--os',
+      (Var("host_os")),
```

Comment:
This doesn't properly handle Apple Silicon as a `host_os`, since it relies on `target_cpu` as well. Not sure if `target_cpu` is set by default, but it should be? If that's the case, then os should be changed to:
`['mac-x64', 'mac-arm64','win-x64','linux-x64']` and mapped to the appropriate downloads

Reply:
Not that it particularly matters right now, since there's no LibreOfficeKit build for macOS anyway :shrug: